docs/content/docs/foundations/starting-workflows.mdx

```diff
@@ -210,6 +210,27 @@ export async function GET(request: Request) {
 }
 ```
 
+### Programmatically list and control runs
+
+If you want to show workflow progress directly in your product UI—or let trusted operators retry or cancel work—you can now call the same administrative helpers the CLI uses, straight from `workflow/api`.
+
+```typescript lineNumbers
+import { listRuns, cancelRun } from 'workflow/api';
```

Comment:
This example should be moved to /docs/api-reference/workflow-api/world and use `getWorld` instead